### PR TITLE
Website content

### PR DESCRIPTION
**Pull request by camachon18**
This is all the code for about 95% of the website. Still need to fix Trade content wording, update README and add favicon for tab label (will do this and update PR, but just wanted to raise this so you guys can have a look). Unit tests all done, mainly checking if content renders since only static page. Ignore the wacky branch name, thought I was going to implement this in different branches. Let me know what you guys think!

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -19,7 +19,54 @@
...
+                        // Add hash (#) to URL when done scrolling (default click behavior)
+                        window.location.hash = hash;
+                    });
+                } // End if
+            });
+        });
+    </script>
+
+    <script>
+        $(function() {
```
Comment (DevinPentecost) [inline public/index.html]: Although this isn't a particularly complicated section of script, it doesn't hurt to have some comments :)

Apparently I don't know how to highlight properly, I am referencing the function below line 56 in index.html, sorry about that.

**Comment by DevinPentecost** [comment]
Hi guys,

Few thoughts:

- Great use of pull requests, although typically you'd want to do each new 'feature' as a new branch. This can be a bit hard to do with the initial commit, but moving forward it can really help with making sure you aren't stepping on each-other's toes.

- What is your unit-test coverage at? Although it can depend on the project, a general guideline is 85% or more. Are you using any continuous integration?

- Do the views need to be named ambiguously? When reviewing the code, I found myself asking 'What is a View1?" I had to frequently refer to other files in the ViewX directories, or the content of the files directly, to discern what it was I was looking at. Something like a "SignUpView" or "AboutUsView" would be easier to parse 'at a glance'. If you do this, just make sure you refactor the names everywhere (if you are using JetBrains IDEs, it should be able to) so your app doesn't break.

- The screenshots and example media are great. Consider adding more to the ReadMe!

- Looks like the code is meeting a consistent standard, which makes it easy to read. I've seen enough poorly written code to really appreciate the effort put in.

-Devin

**Comment by camachon18** [comment]
Hey @DevinPentecost, 

Thanks for reviewing!

Yeah I initially wanted to do each separate feature as a new branch, but I was a bit lazy and just wanted to lump everything into one branch since I was constantly working on several sections of the website at a time and I was the only one working on the website this sprint.

Currently, the code coverage is at about 90% with all but one of our components at full coverage (the one that isn't is something were not planning on implementing this sprint). And we do not have CI set up yet, it would  be a good idea for us to get that set up. Just curious, what are the core benefits of CI? All I know about CI is that it automatically runs all your unit tests whenever you commit/push I think.

Regarding the names, I totally agree that they are ambiguous and this was just a product of rushing. I will refactor those and update the PR. I once heard that one of the things developers struggle with most is naming things....lol

I'll go ahead and make some updates before merging, thanks for the feedback!

File: public/index.html (hunk 1 above)
Comment (camachon18) [reply public/index.html]: Will add comments, good call out!

**Comment by DevinPentecost** [comment]
Hi,

I totally hear you there. Once you get in the flow it will be good to practice git branching - I know that I still need more practice.

That is great code coverage - keep it up. Make sure you hold eachother accountable on unit tests as well, it is really easy to get lazy and not write them and it will _always_ come back to bite you :)

Continuous Integration can be used for a number of things, but automatically running unit tests is a huge benefit. My group's unit tests take over an hour to run, so having a system dedicated to running them and generating reports on them is very useful. It also allows us to know which change caused the tests to start failing, who made the change, and provide insight as to fixing them. I highly recommend it if you have bandwidth. I can always tell, at a glance, how healthy the codebase is.

Developers are notorious for coming up with names that are not descriptive enough... or are **too** descriptive :)

Keep up the great work guys

File: public/index.html (hunk 1 above)
Comment (DevinPentecost) [reply public/index.html]: Just remember, you aren't writing code for a machine, you aren't even writing it for yourself. Write the code for a complete stranger and make sure they won't hate reading it or struggle understanding it. In six months, you will be that stranger :)

**Pull request merged**